pages/common/nano.md

```diff
@@ -7,6 +7,10 @@
 
 `nano {{filename}}`
 
+- Start nano in terminal with {filename} at line {line}, column {column}:
```

Comment:
Let`s remove these curly brackets from the example description. 
Having them only on the example is enough because the example itself makes it pretty intuitive.

Reply:
NP, done (please note there are curly brackets on first example, as well)